mix.exs

```diff
@@ -27,7 +27,7 @@ defmodule OAuth2.Mixfile do
   end
 
   def application do
-    [applications: [:logger, :hackney]]
+    [applications: [:logger, :tesla]]
```

Comment:
```suggestion
    [extra_applications: [:logger]]
```

Reply:
Yeah that work much better was just worried about breaking compatibility.